Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -188,6 +192,8 @@ theorem mul_eq_top : a * b = ∞ ↔ a ≠ 0 ∧ b = ∞ ∨ a = ∞ ∧ b ≠ 0
   WithTop.mul_eq_top_iff
 
 theorem mul_lt_top : a < ∞ → b < ∞ → a * b < ∞ := WithTop.mul_lt_top
+
+@[aesop (rule_sets := [finiteness]) safe apply]
```

Comment:
This isn't `safe`, it's possible that one of the factors is top and the other is `0`. Perhaps this can be `unsafe 75%`?